libs/libarchfpga/src/physical_types.h

```diff
@@ -725,6 +725,9 @@ struct t_physical_tile_type {
     ///@brief Is this t_physical_tile_type an empty type?
     bool is_empty() const;
 
+    ///@brief Returns true if the physical tile type can implement either a .input or .output block type
```

Comment:
This is such a small routine, and can be used in some placement routines, that I suggest inserting the function body here and marking it line.